src/utils.rs

```diff
@@ -15,9 +15,9 @@ pub fn prefix_key(prefix: &dyn AsRef<[u8]>, key: &dyn AsRef<[u8]>) -> Vec<u8> {
     [prefix.as_ref(), key.as_ref()].concat()
 }
 
-/// Calculates the storage fee of an action, given an initial storage amount,
-/// and refunds the predecessor a portion of the attached deposit if necessary.
-/// Returns refund Promise if refund was applied.
```

Comment:
I think plural tense is correct in this case

Reply:
I took a look at a few other doc examples in other libraries including rust docs and simple present tense in plural is the most common style.

I've reverted this change.